packages/create-sitecore-jss/src/templates/angular/package.json

```diff
@@ -124,8 +124,8 @@
     "move-cli": "^1.2.1",
     "npm-run-all": "~4.1.5",
     "protractor": "^7.0.0",
-    "ts-node": "~9.1.1",
-    "typescript": "~4.1.5",
+    "ts-node": "~10.9.0",
+    "typescript": "~4.6.3",
```

Comment:
Here is mismatch between ts version for sitecore-jss-angular, sitecore-jss-angular-schematics and sample app, can we have ~4.7.2 here also?